src/tests/account/test_p256_account.cairo

```diff
@@ -0,0 +1,29 @@
+use openzeppelin::account::interface::P256PublicKey;
```

Comment:
Since we don't have this account in the library yet, I would move this into the test_secp256r1 test file.

Reply:
You do not but we do, see https://github.com/0xknwn/starknet-modular-account/blob/749a0485b0f50b9350f8f41eb1898e88f2f3a195/src/modules/p256validator/p256validator.cairo#L17 . The reason behind this PR is because we use your project as a library to develop a modular account. It makes sense that we use the same tool for secp256k1 and secp256r1 and that you provide the technology. I think if you see you as a library, it really make sense types are kept in files that are not test and Eth and P256 public keys are in a parallel location

Reply:
Got it. I'm not recommending to move the type though, but the contents of this file (to move `test_p256_account.cairo` contents into `test_secp256r1.cairo`).

Reply:
Okay. I will do!